firebase-perf/CHANGELOG.md

```diff
@@ -20,10 +20,36 @@ Refer [GMaven](https://maven.google.com/web/index.html?q=firebase-perf#com.googl
 > **Note:** Refer go/firebase-android-release for `MXX` info.
 
 ## Unreleased
+
+*   {{fixed}} Fixed a bug where screen traces were not capturing frame metrics for multi-activity apps. ([GitHub #3311](//github.com/firebase/firebase-android-sdk/pull/3311))
```

Comment:
You don't have to reference the PR for the fix. We normally just reference the custom issue.

Reply:
We don't have an issue for the FrameMetrics bug, should I reference b/210055697 instead?

Reply:
You don't have to reference anything if it is not a custom issue